scripts/apps/search/services/TagService.ts

```diff
@@ -54,10 +54,24 @@ export function TagService($location, desks, userList, metadata, search,
         };
     }
 
-    function initSelectedParameters(params) {
+    function getParamObject(paramArray: Array<string>) {
+        const key = paramArray[0].trim();
+        // remove parentheses ex: "(two)" becomes "two"
+        const value = paramArray[1].replace(/[()]/g, '');
```

Comment:
Remove only one opening parenthesis if it's the first char in the string and only one closing parenthesis if it's the last char in the string. Current regex removes all parenthesis in the middle as well.